modules/ROOT/pages/mulesoft-support.adoc

```diff
@@ -0,0 +1,13 @@
+= Contact MuleSoft Support
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+For assistance, guidance, professional services customizations, and enhancement requests, please reach out to your MuleSoft Customer Success representative.
+
+To report issues or defects in Anypoint API Community Manager, please submit a support case through the MuleSoft support portal, available at https://help.mulesoft.com .
```

Comment:
Tech docs do not use "please"; GLOBALLY omit
Remove space
```suggestion
To report issues or defects in Anypoint API Community Manager, submit a support case through the MuleSoft support portal, available at https://help.mulesoft.com.
```

Reply:
agree